docs/api_rest.rst

```diff
@@ -0,0 +1,339 @@
+.. _api_rest:
+
+###################
+PyWPS Rest API Doc
+###################
+
+Since version 4.5, PyWPS includes an experimental implementation of the novel OGC API.
+This standard defines the OGC API - Processes API standard.
+This standard builds on the OGC Web Processing Service (WPS) 2.0 Standard
+and defines the processing interface to communicate over a RESTful protocol using JSON encodings.
+
+For more details about the standard please refer to https://github.com/opengeospatial/ogcapi-processes
+
+Defining the input/output format (JSON or XML)
+================================================
+
+WPS 1.0 standard defines input and outputs in XML format.
+WPS 2.0 standard defines input and outputs in JSON format.
```

Comment:
I would not mention WPS 2.0 (that is still xml based ... not ogcapi - processes). 

* WPS 1.0.0: xml
* WPS 2.0.0: xml (https://docs.opengeospatial.org/is/14-065/14-065.html)
* OGC API - Processes: rest-api, json

Reply:
OK, done, thanks!